app/models/service.rb

```diff
@@ -362,8 +362,8 @@ def chargeback_report_name
 
   def generate_chargeback_report(options = {})
     _log.info "Generation of chargeback report for service #{name} started..."
-    MiqReportResult.where(:name => chargeback_report_name).destroy_all
-    report = MiqReport.new(chargeback_yaml)
+    MiqReport.where(:name => chargeback_report_name).destroy_all
+    report = MiqReport.create(chargeback_yaml)
     report.queue_generate_table(options)
```

Comment:
Actually, I think you can avoid persisting the MiqReport instance if you call `report.async_generate_tables(options)` here instead. This will bypass the queue and generate it immediately. I don't think queueing is even necessary here because this method is already running from the queue representing a single report. What do you think @yrudman?